src/mixins/legal-api-mixin.ts

```diff
@@ -117,7 +112,7 @@ export default class LegalApiMixin extends Vue {
 
     if (filingId) {
       // Assign the url business identifier
-      let url = `${this.getBusinessIdentifier}/filings/${filingId}`
+      let url = `/${this.getBusinessIdentifier}/filings/${filingId}`
```

Comment:
I just realized (after chatting with Katie) that there are separate endpoints to post/put/get a business and then to post the completed filing. Can we make this more clear in this code via extra comments or something?

What would help is renaming createFiling/updateFiling to createBusiness/updateBusiness :)

Reply:
Yes, this certainly is a little confusing. We can definitely discuss this in person! 

I will update the method names to help with clarity, thanks for the info!

The jist of it ( As far as i know )..

We POST to businesses using our NR identifier = We receive a filingId in return ( therefore a creating a filing in a sense ). This is essentially a draft filing at this point, as it isn't complete until we do the PUT to the actual filings endpoint with the filingId.

Whenever we go to File and pay ( `completeFiling()` ) it will always save the filing first to `/business/identifier` first. 

So i could be wrong here, but i believe after the initial POST we just need to PUT.

Reply:
Thanks for the explanation. I get it now.

When we continue work on this, we (you or I) should update the comments to better describe what's going on. But for now let's get this code in.